src/core/lv_indev.c

```diff
@@ -816,6 +818,8 @@ static void indev_button_proc(lv_indev_t * i, lv_indev_data_t * data)
  */
 static void indev_proc_press(_lv_indev_proc_t * proc)
 {
+    uint8_t indev_flag = false;
```

Comment:
What about a more descriptive name?
```suggestion
    bool no_move = false;
```

Reply:
I can’t think of a better name, so let's use this.